src/dealWizard/stages/partnerDaoStage/partnerDaoStage.html

```diff
@@ -4,7 +4,7 @@
   <stage-wrapper
     title="Partner DAO Details"
     wizard-manager.bind="wizardManager"
-    validate.call="validateInputs()">
+    show-submit.bind="true">
```

Comment:
This seems a bit icky.
Validation logic for submit is handled somewhere else now?

_(Maybe all good, I'm just going from the diff here)_

Reply:
So the `show-submit.bind="true"` line is unrelated to the validation. It just happened that I made this change on this branch. It basically shows submit button instead of proceed button